src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        final int countOfBalls = 3;
```

Comment:
I meant -> `private static final BALLS_COUNT = 3;` you don't have to write it inside of this method, you should add this in the beginning of this class